cypress/integration/users/loans/declare-lost.spec.js

```diff
@@ -0,0 +1,173 @@
+      cy.getLoanTypes({ limit: 1 });
+      cy.getMaterialTypes({ limit: 1 });
+      cy.getInstanceTypes({ limit: 1 });
+      cy.getLocations({ limit: 1 });
+      cy.getHoldingTypes({ limit: 1 });
+      cy.getHoldingSources({ limit: 1 });
+    }).then(() => {
+      cy.createTempUser([
+        permissions.uiUsersViewLoans.gui,
+        permissions.uiUsersDeclareItemLost.gui,
+      ]).then(({
+        username,
+        password,
+        userId,
+        barcode,
+      }) => {
+        Cypress.env('testUserId', userId);
+
+        cy.addServicePointToUser(Cypress.env('servicePoints')[0].id, userId).then(() => {
+          cy.getServicePointsApi().then((servicePoints) => {
+            const servicePointOwner = servicePoints.map(({
+              id,
+              name,
+            }) => ({
+              value: id,
+              label: name,
+            }));
+
+            cy.createOwnerApi({
+              ...newOwnerData,
+              servicePointOwner,
+            });
+          });
+
+          cy.createInstance({
+            instance: {
+              instanceTypeId: Cypress.env('instanceTypes')[0].id,
+              title: getTestEntityValue(),
+            },
+            holdings: [{
+              holdingsTypeId: Cypress.env('holdingsTypes')[0].id,
+              permanentLocationId: Cypress.env('locations')[0].id,
+              sourceId: Cypress.env('holdingSources')[0].id,
+            }],
+            items: [
+              [{
+                ...newFirstItemData,
+                permanentLoanType: { id: Cypress.env('loanTypes')[0].id },
+                materialType: { id: Cypress.env('materialTypes')[0].id },
+              }, {
+                ...newSecondItemData,
+                permanentLoanType: { id: Cypress.env('loanTypes')[0].id },
+                materialType: { id: Cypress.env('materialTypes')[0].id },
+              }],
+            ],
+          }).then(() => {
+            cy.createItemCheckout({
+              itemBarcode: newFirstItemData.barcode,
+              userBarcode: barcode,
+              servicePointId: Cypress.env('servicePoints')[0].id,
+            });
+            cy.createItemCheckout({
+              itemBarcode: newSecondItemData.barcode,
+              userBarcode: barcode,
+              servicePointId: Cypress.env('servicePoints')[0].id,
+            });
+
+            cy.login(username, password);
+
+            cy.visit(UsersNavigation.getOpenLoansPath(userId));
+          });
+        });
+      });
+    });
+  });
+
+  it('C9191: Declare lost', { tags: [TestType.smoke] }, () => {
+    UsersCard.getApi(Cypress.env('testUserId')).then((user) => {
+      Loans.checkStatusCheckedOut(FIRST_LOAN_ROW_INDEX);
+      Loans.startDeclareLost(FIRST_LOAN_ROW_INDEX);
+      Loans.cancelDeclareLost();
+      Loans.checkStatusCheckedOut(FIRST_LOAN_ROW_INDEX);
+
+      Loans.checkStatusCheckedOut(SECOND_LOAN_ROW_INDEX);
```

Comment:
why do you use SECOND_LOAN? Declare Lost to First loan was canceled. In this case this check is excess.

Reply:
I do it to make sure that second loan has checked out status initially. After that I declare lost second loan and check that its status became declare lost